src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java

```diff
@@ -3,64 +3,47 @@
 import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.*;
 import ru.yandex.practicum.filmorate.model.Film;
 import ru.yandex.practicum.filmorate.service.FilmService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import ru.yandex.practicum.filmorate.exception.ValidationException;
 
 import java.util.Collection;
-import java.util.List;
 
-/**
- * Контроллер для управления фильмами и лайками
- */
 @RestController
 @RequestMapping("/films")
 @Slf4j
 @RequiredArgsConstructor
-@Validated
 public class FilmController {
 
     private final FilmService filmService;
 
-    // Добавление фильма
+    // Добавление фильма — с полной валидацией
     @PostMapping
     public Film createFilm(@Valid @RequestBody Film film) {
+        log.info("Добавление фильма: {}", film);
         return filmService.createFilm(film);
     }
 
-    // Обновление фильма
+    // Обновление фильма — id обязателен, остальные поля опциональны
     @PutMapping
     public Film updateFilm(@Valid @RequestBody Film film) {
```

Comment:
Тогда нужно убрать `@Valid`, чтобы валидация не срабатывала при обновлении

(Then you need to remove `@Valid` so that validation does not trigger on update)